test/test_runner.sh

```diff
@@ -1,8 +1,24 @@
 #!/usr/bin/env bash
```

Comment:
Not sure if this is a new convention or something else, but I've always seen this as ``#!/bin/bash``.